benchbuild/projects/gentoo/xz.py

```diff
@@ -29,23 +29,20 @@ def compile(self):
         download.Wget(test_url, test_archive)
         tar("fxz", test_archive)
 
-    def run_tests(self, runner):
+    def run_tests(self):
         xz = wrapping.wrap(local.path("/usr/bin/xz"), self)
+        xz = run.watch(xz)
```

Comment:
Now there is now underscore

Reply:
Fixed in cc611b4a00e299b1eed8084f79553c9b3259e800